components/Breadcrumbs/BreadcrumbLink.tsx

```diff
@@ -72,14 +72,25 @@ const Explorer: FC<Pick<BaseBreadcrumbLinkProps, 'isCurrent'>> = ({ isCurrent =
   />
 )
 
+const Charts: FC<Pick<BaseBreadcrumbLinkProps, 'isCurrent'>> = ({ isCurrent = false }) => (
+  <BaseBreadcrumbLink
+    to={RoutePaths.Charts}
+    icon={<BlockIcon h="1.25rem" w="1.25rem" color="inherit" />}
+    label="All Charts"
+    isCurrent={isCurrent}
+  />
+)
+
 interface BreadcrumbLinkProps {
   Home: FC<Pick<BaseBreadcrumbLinkProps, 'isCurrent'>>
   Explorer: FC<Pick<BaseBreadcrumbLinkProps, 'isCurrent'>>
+  Charts: FC<Pick<BaseBreadcrumbLinkProps, 'isCurrent'>>
```

Comment:
In another review I suggested that this kind of type be created:
```ts
type CurrentBreadcrumb = <Pick<BaseBreadcrumbLinkProps, 'isCurrent'>
```
And reused across this file